sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -833,6 +833,7 @@ case class CacheTable(
 case class CacheTableAsSelect(
     tempViewName: String,
     plan: LogicalPlan,
+    originalText: Option[String],
```

Comment:
Is any other source of `CacheTableAsSelect` other than `AstBuilder`? Don't we always have this `originalText`?

Reply:
I only see `AstBuilder` as the only source for constructing a `CacheTableAsSelect` so yes we can probably remove the `Option`.